### DISCUSSION TITLE
querying the database: Named pipe connections temporary disabled (see tokio-rs/tokio#3118)

### DISCUSSION THREAD

**Discussion by abii315hussain** [prisma/prisma#13792] [Q&A] [answered]
```
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "prismadb" at "localhost:3306"

Error: Migration engine error:
Error querying the database: Named pipe connections temporary disabled (see tokio-rs/tokio#3118)
```

i just run `npx prisma migrate dev`

.env DB URL
`DATABASE_URL="mysql://root:''@localhost/prismadb?socket=/var/run/mysql/"`

**Comment by janpio** [comment, marked as answer]
Are you on Windows?

**Reply by abii315hussain** [reply]
yes, windows.  I am just trying to connect xamp server to prisma.

**Reply by janpio** [reply]
`tokio-rs/tokio#3118` refers to https://github.com/tokio-rs/tokio/issues/3118, which is an issue in one of our dependencies - but it is closed.

Can you please open a bug report issue for this? Then we can take a look at this. Thanks. (Include as much information as possible how to set up an environment equivalent to yours)

If you enabled non socket communication to that MySQL, you can also use that via a port to connect to MySQL. That definitely works.

**Reply by abii315hussain** [reply]
@janpio Thanks its works via port to connect. I just change my DB URL
`DATABASE_URL="mysql://root:@localhost:3306/prismadb?schema=public`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
